apis/Google.Cloud.Spanner.V1/Google.Cloud.Spanner.V1/TransactionPool.cs

```diff
@@ -243,40 +245,47 @@ public void MarkTransactionUsed()
                 }
             }
 
-            public async Task CreateTransactionAsync(Session session, TransactionOptions options)
+            public Task CreateTransactionAsync(Session session, TransactionOptions options)
```

Comment:
"you typically would not call this method directly simultaneously from different threads" *with the same session*.  Of course you can call with different sessions from different threads.